components/list/test/list.test.js

```diff
@@ -165,6 +165,17 @@ describe('d2l-list', () => {
 			expect(dispatched).to.equal(false);
 		});
 
+		// clean up with flag GAUD-7495-list-interactive-content
+		it('dispatches d2l-list-selection-changes event when interactive element is clicked in selectable item when flag is off', async() => {
+			elem.querySelector('d2l-list-item')._listItemInteractiveEnabled = false;
+			let dispatched = false;
+			const button = elem.querySelector('d2l-button');
+			button.addEventListener('click', () => dispatched = true);
+			setTimeout(() => clickElem(button));
```

Comment:
Clicking doesn't happen immediately, so the timeout shouldn't be necessary.

```suggestion
			clickElem(button);
```